include/eos/morphablemodel/MorphableModel.hpp

```diff
@@ -405,6 +406,17 @@ class MorphableModel
         return landmark_definitions;
     };
 
+    /**
+     * Sets the landmark definitions for this Morphable Model.
+     *
+     * The landmark definitions define mappings from a set of global landmark identifiers, like for
+     * example "eye.right.outer_corner", to the model's respective vertex indices.
+     */
+    void set_landmark_definitions(cpp17::optional<std::unordered_map<std::string, int>> landmark_definitions)
+    {
+        this->landmark_definitions = landmark_definitions;
```

Comment:
I added `this->` because otherwise, the input parameter is setting itself

Reply:
Well-spotted, thanks!